data/filesdescription.yaml

```
@@ -0,0 +1,90 @@
+---
+common:
+  REQUEST-900-EXCLUSION-RULES-BEFORE-CRS.conf.example: >
+    This file is used to add LOCAL exceptions for your site. Often in this file
+    we would see rules that short-circuit inspection and allow certain
+    transactions to skip through inspection.
```

In folded scalar style, an empty line is necessary to insert a break. I think the example should be on its own line.
```suggestion
    transactions to skip through inspection.
    
```